example/src/webhooks.ts

```diff
@@ -16,7 +16,7 @@ void (async function(): Promise<void> {
   const webhooks = await client.fetchWebhooks()
   console.log(`Currently there are ${webhooks.results.length} webhook(s) configured.`)
 
-  let webhook = await client.createWebhook('all', WEBHOOK_URL)
+  let webhook = await client.createWebhook({ event: 'all', url: WEBHOOK_URL })
```

Comment:
Why This change of signature is only a patch version change? Shouldn't it had been at least minor?
I mean because version in package.json is changing from 0.21.0 to 0.21.5

Reply:
You are right, I will merge this PR and force a new release with a minor change :)